basic_wallet_p/wallet/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I was gonna ask. have you look into webpack to setup an React app instead of using create react app.